frontend/src/SectionPage.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Index_Featured_Article_Grid from './Index_Featured_Article_Grid';
+import Advertisement_728x90 from './Advertisement_728x90';
+import NonSports from './NonSports';
+import Section_Features from './Section_Features';
+import './general-style.css';
+
+var hero = {imgUrl:"https://upload.wikimedia.org/wikipedia/commons/thumb/7/79/2010-brown-bear.jpg/200px-2010-brown-bear.jpg",
+			imgAlt:"bear",section:{name:"University News", url:"#"},url:"#",title:"Lorem ipsum dolor",
+			author:{name:"John Doe", url:"#"},date:"May 10, 2010",excerpt:"Lorem ipsum dolor sit amet, elit, sed diam nonummy nibh euismod\
+			tincidunt ut laoreet dolore magna aliquam erat volutpat. Lorem  minim veniam, quis nostrud exerci tation ullamcorper Lorem ipsum\
+			 dolor sit amet, consectetuer [...]"}
+var section = {name:"University News", url:"#", 
+	featured_articles:[{imgUrl:"https://upload.wikimedia.org/wikipedia/commons/thumb/7/79/2010-brown-bear.jpg/200px-2010-brown-bear.jpg", 
+	imgAlt:"bear", url: "#", title:"Sample 1", author: {name: "JD", url:"#"}, date: "today"},
+	{url:"#", title:"Sample 2", author: {name: "JD", url:"#"}, date:"today"},
+	{url:"#", title:"Sample 2", author: {name: "JD", url:"#"}, date:"today"}]};
+
+//props would go in the below - if it's sports, we need to add in that extra thing about sports below (in the space between Section_Features + Adv)
+const SectionPage = (props) => {
+	var section_name = props.match.params.section;
```

Comment:
@seanrayment this is what I mentioned in my e-mail about accessing the parameters from the props to make API requests